src/constants.py

```diff
@@ -10,12 +10,16 @@
 REPLICATION_PASSWORD_KEY = "replication-password"
 REWIND_USER = "rewind"
 REWIND_PASSWORD_KEY = "rewind-password"
+MONITORING_USER = "monitoring"
+MONITORING_PASSWORD_KEY = "monitoring-password"
```

Comment:
Q: should MONITORING_USER be in the list of SYSTEM_USERS?

Reply:
@marceloneppel

Reply:
1a182bc